docs/content/latest/develop/learn/ttl-data-expiration-ycql.md

```diff
@@ -108,6 +108,51 @@ SELECT * FROM pageviews;
 (1 rows)
 ```
 
+## Configuration for time series workloads
+
+Time series workloads frequently use TTL to hold a dataset to a specific size. YCQL includes a file expiration feature optimized for such workloads, reducing both CPU usage and space amplification. This is accomplished by organizing data into files by time, similar to Cassandra's time window compaction strategy.
+
+This feature is available in 2.6.10+, 2.8., and 2.12.1+
+
+#### Configuring for new YCQL time series datasets with TTL
+
+If configuring a new YCQL database for time series datasets and using a default time to live, we recommend the following TServer flag configurations:
+
+##### --tablet_enable_ttl_file_filter = true
+
+Enables expired files to be directly deleted, rather than relying on garbage collection during compaction.
+
+##### --rocksdb_max_file_size_for_compaction = \[the amount of data to be deleted at once, in bytes\]
+
+This value for this flag is chosen depending on how much data is expected to be deleted at once. For example, if a table's TTL is 90 days, it might be desireable to delete three days' worth of data at once. In this case, *rocksdb_max_file_size_for_compaction* should be set to the amount of data expected to be generated in 3 days. Files over this size will be excluded from normal compactions, leading to CPU gains.
+
+Note that there is some tradeoff here between the number of files created and read perforance. A reasonable rule of thumb is to configure the flag such that 30 to 50 files store the full dataset (e.g. 90 days divided by 3 days is 30 files). CPU benefits of using this feature should more than make up for any read performance loss.
+
+##### --sst_files_soft_limit = \[number of expected files at steady state + 20\]
+##### --sst_files_hard_limit = \[number of expected files at steady state + 40\]
+
+The value of these flags is directly dependent on the number of files expected to hold the full dataset at steady state. These flags throttle writes to YCQL if the number of files per tablet exceed their value. Thus, they need to be set in a way that will In the example above, 30 files should hold 90 days worth of data. In this case, *sst_files_soft_limit* would be set to 50, and *sst_files_hard_limit* set to 70.
```

Comment:
The value of these flags depends

Thus, they need to be set in a way that will In the example - that will what? Some text seems to be missing.